components/SideBarInfoIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from 'react'
+import { Flex, useColorMode } from '@chakra-ui/react'
+import { NAMED_COLORS } from 'theme/constants'
+
+interface SideBarInfoIconProps {
+  label: string
+}
+
+const SideBarInfoIcon: FC<SideBarInfoIconProps> = ({ label }) => {
```

Comment:
This component is very similar to Chakra UI component - [`Badge`](https://chakra-ui.com/docs/components/data-display/badge).
May be better to reuse `Badge` component?